core/model-api/src/main/java/org/eclipse/rdf4j/model/base/AbstractLiteral.java

```diff
@@ -584,6 +585,12 @@ static class TemporalAccessorLiteral extends AbstractLiteral {
 
 				.toFormatter();
 
+		private static final DateTimeFormatter DATETIMESTAMP_FORMATTER = new DateTimeFormatterBuilder()
+
+				.appendInstant()
+
+				.toFormatter();
```

Comment:
Are these empty lines deliberate?

Reply:
Can be removed, I tried to follow the same empty lines / grouping as in the rest of the file.

Reply:
Huh yeah, hadn't noticed it was the same for the other files. Odd. I don't particularly think it makes it better or easier to read, but no big deal - happy to leave as-is for now, for consistency's sake at least :)